Wikipedia/Code/DonateFunnel.swift

```diff
@@ -286,4 +290,87 @@ import WMF
 
         logEvent(activeInterface: .articleBanner, action: .impressionSuppressed, actionData: actionData)
     }
+    
+    func logArticleProfile() {
+        logEvent(activeInterface: .articleProfile, action: .profileClick)
+    }
+    
+    func logArticleProfileDonateLoggedIn() {
+        logEvent(activeInterface: .articleProfile, action: .donateStartClick)
+    }
+    
+    func logArticleProfileDonateLoggedOut() {
+        logEvent(activeInterface: .articleProfile, action: .donateStartClick, actionData: ["isAnon": "true"])
```

Comment:
@l-olson1214 Actually an isAnon flag is automatically added deeper in the analytics library ([here](https://github.com/wikimedia/wikipedia-ios/blob/main/WMF%20Framework/Event%20Platform/EventPlatformClient.swift#L523)). So no need to worry about it at the funnel layer. Can you remove these method variants?

Reply:
Interesting!!! Thanks :)